src/IB/IBFEMethod.cpp

```diff
@@ -2859,6 +3505,24 @@ IBFEMethod::spreadTransmissionForceDensity(const int f_data_idx,
 
                     F.zero();
 
+                    // for surface stress normalization term
+                    const double Phi =
+                        Phi_vec ? fe_interp_var_data[qp][Phi_sys_idx][0] : std::numeric_limits<double>::quiet_NaN();
+                    if (Phi_vec)
+                    {
+                        // Compute the value of the first Piola-Kirchhoff stress tensor
+                        // at the quadrature point and add the corresponding forces to
+                        // the right-hand-side vector.
+                        if (d_phi_current_config)
```

Comment:
Same comment as above --- make a variable for the volume scaling and get rid of the `if`-`else` blocks.